shared/components/Link/index.ts

```diff
@@ -0,0 +1,5 @@
+export { Link } from './Link';
+export type { LinkProps, LinkVariant, LinkSize, LinkColor, LinkIconType } from './Link';
```

Comment:
:no_entry_sign: **Must fix**: `LinkColor` is throwing an error. looks like type doesn't exist

Reply:
nice catch, removed